irmaclient/keyshare.go

```diff
@@ -156,7 +155,6 @@ func startKeyshareSession(
 		transport := irma.NewHTTPTransport(scheme.KeyshareServer, !ks.client.Preferences.DeveloperMode)
 		transport.SetHeader(kssUsernameHeader, ks.keyshareServer.Username)
 		transport.SetHeader(kssAuthHeader, ks.keyshareServer.token)
-		transport.SetHeader(kssVersionHeader, "2")
```

Comment:
In my opinion we should already add logic in irmaclient that adds the `/api/v1` when it is not mentioned in the URL. If we then release a version of irmago with the keyshare server change, then the irmaclient counterpart is also able to handle both.

Reply:
I thought of doing that separately as this does not concern the keyshare server but the interaction between the client and what's stated in the scheme.

However, the reason why I like to keep these things small and separate is for a large extend reviewability. So if you, the reviewer, prefers this in one PR, we can add it. As long as we note that these are actually two unrelated issues.

Reply:
Decided jointly we're not implementing the 'replace url' logic in irmaclient this step. It will be handled separate and differently.